### PR TITLE
Increment the SDK version

### PR DESCRIPTION
**Pull request by filizt**
Pump up the SDK version and add changes log

### CODE REVIEW

File: src/SDKs/Blueprint/Management.Blueprint/Microsoft.Azure.Management.Blueprint.csproj [hunk 1]
```diff
@@ -6,17 +6,16 @@
...
     <!-- Remove the Preview description in PackageReleaseNotes once the preview is removed.-->
     <AssemblyTitle>Microsoft Azure Blueprint Management Library</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Management.Blueprint</AssemblyName>
     <PackageTags>Blueprint;</PackageTags>
     <PackageReleaseNotes>
 <![CDATA[This is a preview release sdk for Azure Blueprints, allows you to do Blueprint definition and assignment CRUD.
 
-Changes in 0.12.0-preview:
- - Adding support for excluding AAD principals from blueprint locks 
- - Adding support to retrieve blueprints service SPN objectID
```
Comment (dsgouda) [inline src/SDKs/Blueprint/Management.Blueprint/Microsoft.Azure.Management.Blueprint.csproj]: Please update the `AssemblyVersion` and `AssemblyFileVersion` in the `AssemblyFile.cs` file

**Review by dsgouda — APPROVED** [review]
Looks good apart from the comment

File: src/SDKs/Blueprint/Management.Blueprint/Microsoft.Azure.Management.Blueprint.csproj (hunk 1 above)
Comment (filizt) [reply src/SDKs/Blueprint/Management.Blueprint/Microsoft.Azure.Management.Blueprint.csproj]: Updated.

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
Will merge once CIs pass

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**